homeassistant/components/netatmo/sensor.py

```diff
@@ -447,17 +447,17 @@ def __init__(
                     }
                 )
 
-    @property
-    def available(self) -> bool:
-        """Return entity availability."""
-        return self.state is not None
-
     @callback
     def async_update_callback(self) -> None:
         """Update the entity's state."""
         if (
-            state := getattr(self._module, self.entity_description.netatmo_name)
-        ) is None:
+            not self._module.reachable
+            or (state := getattr(self._module, self.entity_description.netatmo_name))
+            is None
+        ):
+            if self.available:
+                self._attr_available = False
+                self._attr_native_value = None
```

Comment:
We don't need to reset native value I think.

Reply:
Thanks Martin, that is a good point.